src/app/store.ts

```diff
@@ -1,6 +1,9 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { filterSlice } from '../features/filter';
+import { todosSlice } from '../features/todos';
+import { currentTodoSlice } from '../features/currentTodo';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices(filterSlice, todosSlice, currentTodoSlice);
```

Comment:
There is an error here: `combineSlices` should be `combineReducers`. The `combineReducers` function is used to combine multiple slice reducers into a single root reducer. Make sure to import `combineReducers` from `@reduxjs/toolkit`.